test/factories/editionable_worldwide_organisations.rb

```diff
@@ -48,6 +48,12 @@
       end
     end
 
+    trait(:with_french_page) do
```

Comment:
This commit adding the extra test can be merged into the one where the application code was added, since the tests are related to that code.